usort/api.py

```diff
@@ -0,0 +1,93 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import sys
+from pathlib import Path
+from typing import Iterable, Optional, Tuple
+
+from .config import Config
+from .sorting import ImportSortingTransformer
+from .types import Result
+from .util import try_parse, timed, walk
```

Comment:
It's probably worth defining an `__all__` to avoid things like `Path` or `timed` sneaking in if someone does a star-import